cartridges/app_custom_storefront/cartridges/app_custom_storefront/cartridge/controllers/Hello.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+var server = require('server');
+var cache = require('*/cartridge/scripts/middleware/cache');
+
+server.get(cache.applyDefaultCache);
```

Comment:
What is the purpose of this controller? You did not declare a route and there is no result of invoking it (nor json, nor isml)

Reply:
Fixed!